packages/alert-message-react/src/AlertMessage.test.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { cleanup, render } from "@testing-library/react";
+import { InfoAlertMessage, ErrorAlertMessage, WarningAlertMessage, SuccessAlertMessage } from ".";
+import { axe } from "jest-axe";
+
+afterEach(cleanup);
```

Comment:
Denne trengs ikke lengre.

```suggestion
```

Reply:
Hvorfor ikke? Vil man ikke kunne få false positives om det ikke er gjort `cleanup` mellom render i `should render message content`?

Reply:
Nei. Dette var en qurik i tidlige utgaver av RTL. Er ikke nødvendig fra typ RTL 3. Eneste unntaket er om man bruker ReactPureRenderer eller noe sånt, dvs du tar over ansvaret og gjør custom react rendring, noe vi ikke gjør. Så denne kan/bør fjernes fra all the things.

![image](https://user-images.githubusercontent.com/46530368/81150286-eb542400-8f7f-11ea-84a7-774dd1c2c0c3.png)

ref: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library

Reply:
Ty for oppklaring :)